shell/src/main/java/alluxio/cli/validation/hdfs/SecureHdfsValidationTask.java

```diff
@@ -51,6 +50,13 @@
       PRINCIPAL_MAP_MASTER_KEY, PropertyKey.MASTER_KEYTAB_KEY_FILE,
       PRINCIPAL_MAP_WORKER_KEY, PropertyKey.WORKER_KEYTAB_FILE);
 
+  private static final String HDFS_AUTHORIZATION_KEY = "hadoop.security.authorization";
```

Comment:
I'm a bit concerned this introduces `hadoop-common` to shell/ and might lead to us shading jars to not break the downstream. @LuQQiu What do you think? If it's okay we add hadoop common directly to `shell`, i'll make this change. Otherwise I prefer to leave a TODO here and revisit it after 2.3.

Reply:
Discussed with @LuQQiu. Since there's no quick way to verify if we have to shade it in shell/, I added a TODO.